docs/guides/lists.md

```diff
@@ -0,0 +1,112 @@
+# Lists
+
+When you read or create resources, they get added to the `resources` object
+within your state slice. This is useful for quick lookups of a single
+resource, but there are a few things this doesn't let you do:
+
+1. keep track of resource ordering
+2. manage subsets of resources
```

Comment:
nit: the `2.` can actually be `1.` as markdown maintains the OL for you so any future points you don't have to worry about

Reply:
I don't feel strongly either way. I may have left it this way due to the way the site renders, but I'm not sure. If we do change this, we should do every list all at once, and probably in another PR.